### DISCUSSION TITLE
From Express to Hyper-Express

### DISCUSSION THREAD

**Discussion by lodembeep** [quasarframework/quasar#15642] [Ideas / Proposals]
I think it would be a good idea to switch from Express to [Hyper-Express](https://github.com/kartikk221/hyper-express).

I am planing to use it throughout my backend and APIs, i still need to benchmark it but the feedbacks and existing benchmarks are really good. The performance gain could be pretty big. (refer to their repository)

**Comment by TobyMosque** [comment]
https://quasar.dev/quasar-cli-vite/developing-ssr/ssr-webserver#replacing-express-js

**Comment by lodembeep** [comment]
Thanks, but i was thinking about quasar-cli webpack. I've seen the SSR one.
Besides, hyper-express is not fully compatible with express.

**Reply by TobyMosque** [reply]
that is compatible with connect?

**Reply by lodembeep** [reply]
I know for sure that engines are not supported for now, not sure for the rest but the basics should be compatible.
I would love to give it a try, but i am not sure where to start.

**Reply by hinogi** [reply]
Does this mean you could also use nuxt under the hood?

**Reply by lodembeep** [reply]
No idea honestly
